heartbeat/galera

```diff
@@ -954,7 +954,7 @@ if [ -z "${OCF_RESKEY_check_user}" ]; then
 fi
 : ${OCF_RESKEY_check_user="root"}
 
-MYSQL_OPTIONS_CHECK="-nNE --user=${OCF_RESKEY_check_user}"
+MYSQL_OPTIONS_CHECK="--no-defaults -nNE --user=${OCF_RESKEY_check_user}"
```

Comment:
I think that would break cases where the user instantiated the resource without specifying OCF_RESKEY_check_user and OCF_RESKEY_check_passwd (or relying on /etc/sysconfig/clustercheck) and relies on the contents of /root/.my.cnf to work.
Maybe we should add an option e.g. OCF_RESKEY_no_defaults that would use that flag, and keep the default as is for backward compatibility?

Reply:
Hmm, yeah, adding `--no-defaults` is definitely a breaking change there..